src/Form/FieldTextarea.js

```diff
@@ -5,30 +5,94 @@ import React from 'react';
 
 import FieldInput from './FieldInput';
 import IconEdit from './icons/IconEdit';
+import Util from '../Util/Util';
+
+class FieldTextarea extends FieldInput {
+  constructor() {
+    super(...arguments);
+
+    this.state = {height: 0};
+
+    this.updateTextareaHeight = Util.throttle(this.updateTextareaHeight, 100);
+    this.handleContentEditableBlur = this.handleContentEditableBlur.bind(this);
+    this.handleContentEditableChange =
+      this.handleContentEditableChange.bind(this);
+    this.handleContentEditableFocus =
+      this.handleContentEditableFocus.bind(this);
+  }
+
+  componentWillMount() {
+    this.setState({height: this.props.minHeight});
```

Comment:
Done! I was thinking props aren't available in the constructor. But they are....